docs/cheat-sheet.md

```diff
@@ -0,0 +1,36 @@
+# Cheat sheet
```

Comment:
@joelochlann should we move this into the runbook? Thinking about it there's a strong argument to start a `docs` folder in this repo with explainers, cheat sheets and tips per service

Reply:
yeah anything here that's not in runbook already, let's move there